react-app-drawer-component/my-app/src/AppDrawer.tsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from 'react';
+import { DrawerItem } from './DrawerItem';
+import { FaBars } from 'react-icons/fa';
+
+type Props = {
+  heading: string;
+  links: string[];
+  onSelect: (pageHeader: string) => void;
+};
+
+export function AppDrawer({ heading, links, onSelect }: Props) {
+  const [showDrawer, setShowDrawer] = useState(false);
+
+  function handleSelection(pageHeader: string): void {
+    onSelect(pageHeader);
+    setShowDrawer(false);
+  }
+
+  return (
+    <>
+      <div
+        className={`transition-colors ${
```

Comment:
Yes, I agree that Tailwind makes this very verbose! You can always mix CSS classes and Tailwind.